### DISCUSSION TITLE
[Feature] Remove Duplicates - With smart search

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#10232] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Remove Duplicates is the start of something great, seriously even Google isn't there yet, it's amazing and I want to say thank you to everyone who made it happen!

My suggestion is to add filters of the smart search to the duplicate identifier, in this way there will be more options to choose which duplicates to exclude or include.

For example, if I want all the photos with my face that have duplicates not to be displayed in the duplicate viewer, I can mark my face as an option, or for example sorting duplicates according to the context or exclusion of the context, etc.

In addition, filtering of live photos, videos, in album or not... that would be very helpful

Also the possibility to exclude a certain model of camera, for professional photographers for whom every image is important even if it looks the same to machine learning.

In other words, to make the smart search work with the duplicate viewer.

By the way, I would be really happy to see an option for a button that deletes the duplicates in which there is no face, which would be very useful because if we delete a duplicate image that looks similar but is recognized by the face recognition and the other image is not, we are actually losing something here rather than something.

A kind of duplication of https://github.com/immich-app/immich/discussions/10163 But not really
### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mertalev** [comment]
It'd be easy to add an "is duplicate" checkbox for advanced search. It wouldn't actually group by duplicates, but would that still be helpful?

**Reply by aviv926** [reply]
My intention was to bring the smart search filters into the duplicate viewer to allow better control of what is displayed there.

![image](https://github.com/immich-app/immich/assets/51673860/efc6bcd3-eb05-40b8-9f7e-3269011830ab)

**Reply by mertalev** [reply]
I feel like a search bar is overkill for a few images. Maybe if we changed the page to show more than one group of assets at a time.

**Reply by aviv926** [reply]
Maybe the search bar itself is a bit overkill, but in my opinion the smart search filter options can be an improvement that will make sorting easier, especially when it works on external libraries

**Reply by mertalev** [reply]
That's true, some of the filters can be useful
